rust/kcl-python-bindings/src/lib.rs

```diff
@@ -356,6 +357,64 @@ async fn mock_execute(path: String) -> PyResult<bool> {
         .map_err(|err| pyo3::exceptions::PyException::new_err(err.to_string()))?
 }
 
+#[pyfunction]
+async fn import_and_snapshot(path: String, format: InputFormat3d, image_format: ImageFormat) -> PyResult<Vec<u8>> {
+    let img = import_and_snapshot_views(vec![path], format, image_format, Vec::new())
+        .await?
+        .pop();
+    Ok(img.unwrap())
+}
+
+fn to_py_exception(err: impl std::fmt::Display) -> PyErr {
+    PyException::new_err(err.to_string())
+}
+
+#[pyfunction]
+async fn import_and_snapshot_views(
+    filepaths: Vec<String>,
```

Comment:
Interesting choice to input multiple filepaths. I think we will only need to input one at a time. I'm curious about the intention here. Is the idea to import multiple models into the same scene and snapshot them together?

Reply:
Some CAD projects can be multiple files and import parts from each other. This is why the import API allows you to send multiple files. If you only want to send 1, that will work fine too!